.buildkite/ftr_configs.yml

```diff
@@ -460,10 +460,7 @@ enabled:
   - x-pack/test/security_solution_api_integration/test_suites/detections_response/default_license/rule_creation/configs/ess.config.ts
   - x-pack/test/security_solution_api_integration/test_suites/detections_response/default_license/actions/configs/serverless.config.ts
   - x-pack/test/security_solution_api_integration/test_suites/detections_response/default_license/actions/configs/ess.config.ts
+  - x-pack/test/security_solution_api_integration/test_suites/detections_response/basic_license/rule_management/configs/serverless.config.ts
```

Comment:
Remember to add:
```
x-pack/test/security_solution_api_integration/test_suites/detections_response/basic_license/rule_management
```
to the **CODEOWNERS** file and assign it to our team. (And any more folders if you add them).
[See this PR for reference.](https://github.com/elastic/kibana/pull/169951)